src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Extensions/FileExtensionTests.cs

```diff
@@ -225,60 +228,25 @@ public void ToFileInfo_BlobWithNoNameKey()
                 meta: new Dictionary<string, string>()
             ));
 
-            Assert.Equal(_file.Id, result.Id);
+            Assert.Equal(_releaseFile.FileId, result.Id);
             Assert.Equal("pdf", result.Extension);
             Assert.Equal("ancillary.pdf", result.FileName);
             Assert.Equal("ancillary.pdf", result.Name);
-            Assert.Equal(_file.Path(), result.Path);
             Assert.Equal("400 B", result.Size);
             Assert.Equal(Ancillary, result.Type);
         }
 
```

Comment:
I don't think anythings been added to test `ToFileInfo(this MethodologyFile)`. That should be added to MethodologyFileExtensionTests